.github/actions/setup-linux/action.yml

```diff
@@ -51,22 +51,21 @@ runs:
         # Note: the CI machine does not meet CUDA 13's driver requirement.
         # Compatibility matrix:
         # https://docs.nvidia.com/deeplearning/cudnn/backend/latest/reference/support-matrix.html
-        # The `nvcc` is installed into `/usr/local/cuda-VERSION/bin/nvcc` - but
-        # it's *not* on the default toolkit path.
         PACKAGES: |
           {
             "cuda-12.6": "libcudnn9-dev-cuda-12 cuda-toolkit-12-6",
             "cuda-12.9": "libcudnn9-dev-cuda-12 cuda-toolkit-12-9",
             "cuda-13.0": "libcudnn9-dev-cuda-13 cuda-toolkit-13-0"
           }
       run: |
-        export ARCH=${{ runner.arch == 'arm64' && 'arm64' || 'x86_64' }}
+        export ARCH=${{ runner.arch == 'arm64' && 'sbsa' || 'x86_64' }}
```

Comment:
Should we leave a comment here why it’s sbsa instead of arm64 or is that well understood by anybody working in this space?